### DISCUSSION TITLE
Execute w/ Limited User Permissions?

### DISCUSSION THREAD

**Discussion by AlmightyJojo** [Rem0o/FanControl.Releases#1237] [Q&A]
Hi - Great software!

Anyway to bypass UAC and allow Fan control to work on business domain PCs which lack administrative rights?
I've tried few methods to execute it with task scheduler but no go.

Any suggestions aside from granting admin permissions or disabling UAC?
Thanks!

Feature request:  Windows service which run as system account rather than user account?

**Comment by IslandJohn** [comment]
I would also like to see this. Example: Corsair iCUE supports it.
